main.tf

```diff
@@ -4,27 +4,12 @@
 # version:  1.11.0
 ##-----------------------------------------------------------------------------
 
-##-----------------------------------------------------------------------------
-# Terraform
-terraform {
```

Comment:
Are you replacing this anywhere else, please? We should keep some basic `required_providers` block (although without the `this` alias if not needed).